content/en/docs/best-practices/general-principles/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: General Principles
+description: The general principles for your Istio deployments.
+weight: 1
+icon: best-practices
+---
+
+We've identified the following general principles to help you get the most out
+of your Istio deployments.
+
+## Make your deployment simple
+
+When choosing a [deployment model](/docs/concepts/deployment-models),
+**prefer simplicity over complexity** wherever possible.
+
+## Make your deployment fast
+
+Include clusters in your deployment across the globe for **geographic
+proximity to end-users**. The proximity helps your deployment have low latency.
+
+## Make your deployment reliable
+
+Include clusters in your deployment **across multiple availability regions
+and zones** within each region. This approach helps you avoid global failures.
+
+Follow these recommendations to enhance the reliability of your deployment:
+
+- Deploy each workload to at least one cluster per region.
+- Deploy a control plane to at least one cluster per region.
+- Ensure each control plane has a dedicated configuration source.
+- Roll out changes incrementally. Do not change all failure domains at once.
```

Comment:
Change one failure domain at a time? A subset of failure domains? Not sure what best practice actually is, what does the SRE book say?  :)

Reply:
```suggestion
- Roll out Istio and Kubernetes configuration changes incrementally, one failure domain at a time.
```

Reply:
Can we make something a best practice if we don't actually support it today? There is not really any (good) way to roll things out incrementally with Istio today

Reply:
@howardjohn that's a good point. What are folks doing out in the wild for this? They must make this work somehow, no?

Reply:
Fingers crossed things work :slightly_smiling_face: 

Generally people are doing canaries by cluster, and upgrading on a dev cluster